frontend/src/routes/(authenticated)/project/[project_code]/BulkAddProjectMembers.svelte

```diff
@@ -0,0 +1,151 @@
+<script lang="ts">
+  import { BadgeButton, MemberBadge } from '$lib/components/Badges';
+  import { DialogResponse, FormModal } from '$lib/components/modals';
+  import { Input, TextArea, passwordFormRules } from '$lib/forms';
+  import { ProjectRole, type BulkAddProjectMembersResult } from '$lib/gql/types';
+  import t from '$lib/i18n';
+  import { z } from 'zod';
+  import { _bulkAddProjectMembers } from './+page';
+  import { hash } from '$lib/util/hash';
+  import { AdminContent } from '$lib/layout';
+  import Icon from '$lib/icons/Icon.svelte';
+  import BadgeList from '$lib/components/Badges/BadgeList.svelte';
+  import { distinct } from '$lib/util/array';
+
+  enum BulkAddSteps {
+    Add,
+    Results,
+  }
+
+  let currentStep = BulkAddSteps.Add;
+
+  export let projectId: string;
+  const schema = z.object({
+    usernamesText: z.string().min(1, $t('register.name_missing')),
+    password: passwordFormRules($t),
+  });
+
+  let formModal: FormModal<typeof schema>;
+  $: form = formModal?.form();
+
+  let addedMembers: BulkAddProjectMembersResult['addedMembers'] = [{username: 'test2', 'role': ProjectRole.Manager}];
+  let createdMembers: BulkAddProjectMembersResult['createdMembers'] = [{username: 'test', 'role': ProjectRole.Editor}];
+  let existingMembers: BulkAddProjectMembersResult['existingMembers'] = [{username: 'test2', 'role': ProjectRole.Manager}];
```

Comment:
let's remove the test data here so we don't accidentally show it to users through a bug.

Reply:
Didn't recognize that test data, turns out that's because it's from Tim's changes. So no wonder I missed it earlier: it wasn't there. :grinning: Anyway, done in commit bd241caf38e5cc3c549f3ad2599eb0a1d4d88d46.

Reply:
🤦